specification/automation/resource-manager/readme.md

```diff
@@ -25,7 +25,7 @@ These are the global settings for the Automation API.
 title: AutomationClient
 description: Automation Client
 openapi-type: arm
-tag: package-2021-06-22
+tag: package-2022-01-31
```

Comment:
Can I know why change this tag?

Reply:
I changed the tag, because I thought the tag should point to the most recent versions. I will revert this just to keep the PR based on xms ids.